package/kedro_viz/api/rest/responses/pipelines.py

```diff
@@ -238,10 +238,17 @@ def get_pipeline_response(
     )
 
 
-def get_kedro_project_json_data():
-    """Decodes the default response and returns the Kedro project JSON data.
+def get_kedro_project_json_data(pipeline_name: Optional[str] = None):
+    """Decodes the default response for the specified pipeline_name
```

Comment:
nit - Decodes the response for the specified pipeline_name  and returns the Kedro project JSON data.